macosx/kxac97/Ac97App.mm

```diff
@@ -87,26 +88,48 @@ - (void)tabView:(NSTabView *)tabView didSelectTabViewItem:(NSTabViewItem *)item
 
 - (NSApplicationTerminateReply)applicationShouldTerminate:(NSApplication *)sender
 {
-    for (NSTabViewItem *item in [tabView tabViewItems])
+//    for (NSTabViewItem *item in [tabView tabViewItems])
+//    {
+//        int device = [[item identifier] intValue];
+//
+//        if ([ac97 kx] && ([ac97 kx]->get_device_num() == device))
+//            [self saveState:[ac97 kx]];
+//        else
+//        {
+//            iKX *kx = iKX::create(device);
+//            if (kx)
+//            {
+//                [self saveState:kx];
+//                delete kx;
+//            }
+//        }
+//    }
+    
+    for (Byte i = 0; i < nItem; i++)
```

Comment:
nItem is always '0', need to rewrite the code to fix the problem.

Reply:
Forgot insert the increment. nItem++;